frontend/src/components/pages/login/Login.tsx

```diff
@@ -6,10 +6,20 @@ import Footer from '../../shared/footer/Footer';
 import Tile from '../../shared/tile/Tile';
 import Box from '@material-ui/core/Box';
 import { Link } from 'react-router-dom';
+import { GoogleLogin } from 'react-google-login';
+import axios from 'axios';
 import user from '../../mocks/user-mock.json';
 
 const saveUserToStorage = () => {
   localStorage.setItem('loginedUser', JSON.stringify(user));
+};
+function responseGoogle(response: any): void {
+  axios({
+    method: 'POST',
+    url: 'http://localhost:8000/api/googleLogin',
```

Comment:
http://localhost:8000 is hardcode value - please extract ot the dynamic window object property